### PR TITLE
Fixed minor developer docs issues in recently added docs

### PR DESCRIPTION
**Pull request by jacoswan**
Fixed a few duplicate titles and encoding issues in the docs that were recently added via public resources.

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Closes # .

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Removed a few duplicate titles and corrected ``` to `'` to fix encoding issues. 
2. Most significant change that should be checked is to `template_structure.md` where a good portion of the content was updated and an attempt was made to fix the long table that references old versions of WooCommerce templates, which is broken in the currently-live version. 

### Changelog entry

-   [x] Automatically create a changelog entry from the details below.

-   [ ] This Pull Request does not require a changelog entry. (Comment required below)

<details>

<summary>Changelog Entry Details</summary>

#### Significance

-   [ ] Patch
-   [x] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [x] Enhancement - Improvement to existing functionality

#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>

### CODE REVIEW

**Review by shaybanshee — DISMISSED** [review]
Thanks @jacoswan ! Updating: There are some linting issues we need to clear first . 

> >>>Linting failed for file: docs/theme-development/template-structure.md <<<
stdin:6 MD036/no-emphasis-as-heading Emphasis used instead of a heading [Context: "NOTE"]
stdin:8 MD003/heading-style Heading style [Expected: atx; Actual: setext]
stdin:139 MD036/no-emphasis-as-heading Emphasis used instead of a heading [Context: "Note"]
stdin:141 MD003/heading-style Heading style [Expected: atx; Actual: setext]
stdin:141:382 MD026/no-trailing-punctuation Trailing punctuation in heading [Punctuation: '.']

And changelog: 
`Run pnpm utils changefile 50723 -o woocommerce

> woocommerce-monorepo@ utils /home/runner/work/woocommerce/woocommerce
> ./tools/monorepo-utils/bin/run "changefile" "50723" "-o" "woocommerce"

Getting pull request data for PR number 50723
Task complete.
No changelog message or comment found
 ELIFECYCLE  Command failed with exit code 1.
 WARN   Local package.json exists, but node_modules missing, did you mean to install?
Error: Process completed with exit code 1.`

File: docs/theme-development/template-structure.md [hunk 1]
```diff
@@ -3,121 +3,149 @@ post_title: Template structure & Overriding templates via a theme
 ---
 
 ---
+**NOTE**
```
Comment (piinthecloud) [inline docs/theme-development/template-structure.md:6]: Revert to:

> **Note:** this document is geared toward template development for classic themes. Check this other document for [block theme development](../../plugins/woocommerce-blocks/docs/designers/theming/README.md).

File: docs/theme-development/template-structure.md [hunk 2]
```diff
@@ -3,121 +3,149 @@ post_title: Template structure & Overriding templates via a theme
...
+To override the admin order notification in our example, copy `wp-content/plugins/woocommerce/templates/emails/admin-new-order.php` to `wp-content/themes/storefront-child/woocommerce/emails/admin-new-order.php`
+
+The copied file will now override the WooCommerce default template file, so you can make any changes you wish to the copied file, and see it reflected in the resulting output.
 
 ---
+**Note**
 
-## How to Edit Files
-
+A (desirable) side-effect of your templates being upgrade-safe is that WooCommerce core templates will update, but your custom overrides will not. You may occassionally see notices in your System Status report that says, e.g. “version 3.5.0 is out of date. The core version is 3.7.0″. Should that happen, follow the Fixing Outdated WooCommerce Templates guide to bring them in line.
```
Comment (piinthecloud) [inline docs/theme-development/template-structure.md:141]: Update to:

> **Note**: A (desirable) side-effect of your templates being upgrade-safe is that WooCommerce core templates will update, but your custom overrides will not. You may occassionally see notices in your System Status report that says, e.g. “version 3.5.0 is out of date. The core version is 3.7.0″. Should that happen, follow the Fixing Outdated WooCommerce Templates guide to bring them in line.

**Comment by shaybanshee** [comment]
There was an issue with the changelog coming from the initial fork - going to review the contribution guidelines to help clarify this issue, and create a more comprehensive contribution guide. @piinthecloud created a new branch with these changes which has already been merged, closing this PR.

**Pull request closed without merging**